src/cmd/cli/command/version.go

```diff
@@ -53,9 +60,14 @@ func GetLatestVersion(ctx context.Context) (string, error) {
 	}
 	defer resp.Body.Close()
 	if resp.StatusCode != 200 {
+		term.Debug(resp.Header)
```

Comment:
You've seen this doc, right?

https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#checking-the-status-of-your-rate-limit

You should be able to verify if the rate limit is in play if either the `x-ratelimit-remaining` header is `0` or if `retry-after` is not `nil`.

If the the `x-ratelimit-remaining` header is `0`, then you can check the `x-ratelimit-reset` to see the number of seconds the client should wait.

Reply:
There's many error conditions so I don't want to make this specific to throttling.